src/components/home/home.tsx

```diff
@@ -10,8 +10,62 @@ import {
   Form,
   Row,
 } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+import { getDatasetsSearchResp } from "../../api/browse";
+import { searchResponseModel } from "../../models/dataset";
+import { facetFilterModel, facetModel } from "../../models/facets";
 
 const Home = () => {
+  let navigate = useNavigate();
+
+  const [searchResults, setSearchResults] =
+    React.useState<searchResponseModel | null>(null);
+
+  const [filterDict, setFilterDict] = React.useState<facetFilterModel[]>([]);
+
+  React.useEffect(
+    () => {
+      const getData = () => {
+        getDatasetsSearchResp(setSearchResults, [], "", 0, 1);
+      };
+      getData();
+    }, // eslint-disable-next-line react-hooks/exhaustive-deps
+    []
+  );
+
+  var facetList: facetModel[] | null = null;
+  var dsCount: number = 0;
+
+  if (searchResults !== null) {
+    if (searchResults.hits.length > 0 || searchResults.count === -1) {
+      facetList = searchResults.facets;
+      dsCount = searchResults.count;
+    } else {
+      facetList = [];
+      dsCount = 0;
+    }
+  }
+
+  const searchDatasets = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (filterDict.length > 0) {
+      let filterURI = filterDict.map((x) => x.key + ":" + x.value).join(";");
+      navigate("/browse?q=" + searchKeyword + "&f=" + filterURI);
```

Comment:
Check for empty search keyword and if true, do not add "q" to the search params in URL